src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx

```diff
@@ -124,14 +121,13 @@ export default function PublicationDetailsForm({
                     />
                   )}
 
-                  {canUpdatePublication && themes && initialValues?.topicId && (
-                    <FormFieldThemeTopicSelect<FormValues>
+                  {canUpdatePublication && themes && (
+                    <FormFieldThemeSelect<FormValues>
                       id={id}
                       inline={false}
-                      legend="Choose a topic for this publication"
+                      legend="Choose a theme for this publication"
                       legendHidden
-                      name="topicId"
-                      themeInputName="themeId"
+                      name="themeId"
                       themes={themes}
                     />
                   )}
```

Comment:
Think we can just use a typical FormFieldSelect directly here and do away with the "Theme" specific FormSelect/FormFieldSelect components (`*-admin/src/components/form/FormFieldThemeSelect.tsx`, & `.../FormThemeSelect.tsx` and associated .module.scss files)

Here's a replacement form select that should be able to be dropped in instead:
```jsx
{canUpdatePublication && themes && (
                    <FormFieldSelect<FormValues>
                      className="govuk-!-width-one-half"
                      id={id}
                      inline={false}
                      label="Choose a theme for this publication"
                      name="themeId"
                      options={themes.map(theme => {
                        return {
                          label: theme.title,
                          value: theme.id,
                        };
                      })}
                    />
                  )}
```